ui/infra-compare/InfraCompare.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import withValidation from '../perfherder/Validation';
+
+import { getCounterMap } from './helpers';
+import { phTimeRanges } from './constants';
+import InfraCompareTableView from './InfraCompareTableView';
+
+class InfraCompareView extends React.PureComponent {
+  getInterval = (oldTimestamp, newTimestamp) => {
+    const now = new Date().getTime() / 1000;
+    let timeRange = Math.min(oldTimestamp, newTimestamp);
+    timeRange = Math.round(now - timeRange);
+    const newTimeRange = phTimeRanges.find((time) => timeRange <= time.value);
+    return newTimeRange.value;
+  };
+
+  queryParams = (project, interval) => ({
```

Comment:
I'm not sure I understand the point of this function.  Seems like it's just converting some parameters into an object.